batch/batch/driver/instance.py

```diff
@@ -68,11 +75,11 @@ async def create(app, inst_coll, name, activation_token, instance_config: Instan
                 now,
                 now,
                 INSTANCE_VERSION,
-                instance_config.location,
+                location,
                 inst_coll.name,
-                instance_config.machine_type,
-                instance_config.preemptible,
-                base64.b64encode(json.dumps(instance_config.config).encode()).decode()
+                machine_type,
+                preemptible,
+                json.dumps(instance_config.to_dict()),
```

Comment:
I'm worried about backwards compatibility here.

Reply:
ah the base64 bit? I should restore that, though I'm not sure why we did that.

Reply:
I thought it was more compact to store.

Reply:
Regarding the null issue, no active instances will have null, so I don't think we need to handle that any more.

Reply:
Yeah, I suppose it should be more compact. Now that we're just storing a few fields I don't think it matters much.